tests/foreman/api/test_provisioning.py

```diff
@@ -98,16 +100,22 @@ def test_rhel_pxe_provisioning(
 
     :parametrized: yes
     """
+    if pxe_loader.vm_firmware == 'bios':
+        pytest.skip('Test cannot be run on BIOS as its not supported')
```

Comment:
For IPv4 is still supported, so I think you missed condition for IPv6 here?

Reply:
Yeah, updated now